source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -211,3 +211,16 @@ sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU=
 You do not need to make any changes to the HTML.
 
 [Learn more about Content Security Policy on the MDN website](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP).
+
+#### If you are not including the hash in your Content Security Policy (CSP)
+
+While we recommend you include the hash in your CSP, we understand this may not always be practical. For example, if you miss a future update to the hash. In this case, updating GOV.UK Frontend could potentially break some JavaScript components. If you're not sure about using the hash, consult your security expert before you decide on a CSP.
+
+If you do not want to include the hash, then you can pass a `scriptNonce` variable to be set on the script instead. However, you should provide a nonce that hostile actors cannot guess. Otherwise, they could easily find a way around your CSP.
```

Comment:
'scriptNonce` variable or `scriptNonce` attribute?

Reply:
To answer "is it unavoidable" I'd say that I'd struggle to find another word which would:

a) describe it concisely (as that's what the attribute is called)
b) be searchable in the code/documentation when people are looking for the feature (as that's what the feature is called)

I would love it if the feature didn't have that name but it's similar to `href` - if you try to avoid that name it can be hard to make sure people know the impact that the variable has.

So my answer is that it's better to follow the name provided to us by the spec, it's not unavoidable but avoiding it would come with potential challenges.